src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+$day-size: 100px;
+$days: ('mon' 0, 'tue' 1, 'wed' 2, 'thu' 3, 'fri' 4, 'sat' 5, 'sun' 6);
+$gap: 1px;
+$padding: 10px;
+
+@import './main';
+
 body {
   margin: 0;
+  box-sizing: border-box;
```

Comment:
The `box-sizing` property should be applied globally to all elements using the universal selector `*`, not just to the `body`. This ensures that all elements on the page are consistently box-sized, which is a common best practice in CSS.